docs/resources/kernel_module.md.erb

```diff
@@ -4,20 +4,62 @@ title: About the kernel_module Resource
 
 # kernel_module
 
-Use the `kernel_module` InSpec audit resource to test kernel modules on Linux platforms. These parameters are located under `/lib/modules`. Any submodule may be tested using this resource.
+Use the `kernel_module` InSpec audit resource to test kernel modules on Linux
+platforms. These parameters are located under `/lib/modules`. Any submodule may
+be tested using this resource.
+
+The `kernel_module` resource can also verify if a kernel module is `blacklisted`
+or if a module is disabled via a fake install using the `bin_true` or `bin_false`
+method.
 
 ## Syntax
 
-A `kernel_module` resource block declares a module name, and then tests if that module is a loadable kernel module:
+A `kernel_module` resource block declares a module name, and then tests if that
+module is a loadable kernel module:
+
+    describe kernel_module('video') do
+      it { should be_loaded }
+      it { should_not be_disabled }
+      it { should_not be_blacklisted }
+    end
 
-    describe kernel_module('module_name') do
+    describe kernel_module('video') do
+      it { should_not be_blacklisted }
       it { should be_loaded }
     end
 
+    # via 'bin_false'
```

Comment:
I think it does not matter if it was blacklisted by `/bin/false`, `/bin/true` or blacklist config, lets simplify the examples